docs-site/content/guide/reference-implementations/kotlin-soccer-search.md

```diff
@@ -0,0 +1,13 @@
+# Using Search with Kotlin Compose and Typesense
+
+This site showcases how to use Typesense with Kotlin Compose to search through a soccer match dataset, implementing custom UI components and leveraging Typesense for fast, faceted search functionality.
+
+[Source Code](https://github.com/DATAGNIKAN/showcase-soccer-matches-search-kotlin-compose)
+
+### Key Highlights
```

Comment:
For each of the links in this section, instead of linking to `blob/main`, could you link to the permalink instead? 

Here's how to get the permalink:

![Screenshot 2024-09-25 at 1 48 45 PM](https://github.com/user-attachments/assets/d6abeb65-a318-4872-8f4a-35214c2cc939)

This will ensure that if changes are made to the repo, the line numbers, etc still stay consistent as they will now point to a particular commit